GEOSldas_App/util/postproc/ObsFcstAna_stats/Get_ObsFcstAna_stats.py

```diff
@@ -69,7 +69,7 @@ def main():
     # The plotting scripts can also run standalone using the individual Plot_stats_*.py scripts,
     #   as long as the monthly sum files are available.
     
-    plot_maps       = False
+    plot_maps       = False 
```

Comment:
I think we need to decide if we want one main script (get monthly sums, plot maps, plot time series) or three main scripts.  As things are now, we're duplicating the plotting of maps and time series, which only confuses things.  I think it would be cleaner to remove the plot_maps and plot_timeseries from this script altogether, and to rename it "Get_ObsFcstAna_sums.py", and then have the Plot*.py files as stand-alone scripts.   (IMPORTANT: If you implement this suggestion, use "**git** mv Get_ObsFcstAna_stats.py Get_ObsFcstAna_sums.py" to retain the history of changes!)